app/views/associate_ucr_summary.scala.html

```diff
@@ -59,7 +59,7 @@ <h2 id="master-consignment">@messages("associate.ucr.summary.masterConsignment")
                     <td id="mucr-type">@messages("associate.ucr.summary.kind.mucr")</td>
                     <td id="mucr-reference">@mucr</td>
```

Comment:
I am aware it is not a part of this change, but should these elements have also their id's changed?

Reply:
I am going to make a note of that when we move to the GDS design system! Hawk Eye